### PR TITLE
Hotfix/wpcs

### PR DESCRIPTION
**Pull request by NicktheGeek**
addresses #7 
@reaktivstudios/reviewers this is ready to review

### CODE REVIEW

**Comment by NicktheGeek** [comment]
this rewrites like 96% of the original file between the automated and manual fixes.

Most of this is white space changes and adding periods to comments.

Also, all `return` statements after a redirect are switched to `exit;` and there were several instances of the wrong escape being used or no escaping of get text functions.

File: gppro-export-css.php [hunk 1]
```diff
@@ -102,139 +100,125 @@ public function gppro_active_check() {
...
 
-		// check for class
+		// Check for class.
 		if ( ! class_exists( 'Genesis_Palette_Pro' ) ) {
-			$failure	= menu_page_url( 'genesis-palette-pro', 0 ).'&section=build_settings&export-css=failure&reason=noclass';
-			wp_safe_redirect( $failure );
-
-			return;
+			$failure = menu_page_url( 'genesis-palette-pro', 0 ) . '&section=build_settings&export-css=failure&reason=noclass';
+			wp_safe_redirect($failure);
```
Comment (jjeaton) [inline gppro-export-css.php]: Looks like there's still a whitespace issue here.

File: gppro-export-css.php [hunk 2]
```diff
@@ -102,139 +100,125 @@ public function gppro_active_check() {
...
-		$file	= Genesis_Palette_Pro::filebase();
+		// Get CSS file.
+		$file = Genesis_Palette_Pro::filebase();
 		if ( ! file_exists( $file['dir'] ) ) {
-			$failure	= menu_page_url( 'genesis-palette-pro', 0 ).'&section=build_settings&export-css=failure&reason=nofile';
-			wp_safe_redirect( $failure );
-
-			return;
+			$failure = menu_page_url( 'genesis-palette-pro', 0 ) . '&section=build_settings&export-css=failure&reason=nofile';
+			wp_safe_redirect($failure);
```
Comment (jjeaton) [inline gppro-export-css.php]: whitespace around parens

**Review by jjeaton — APPROVED** [review]
2 minor wpcs changes

**Pull request merged**